lib/galaxy_test/api/test_folder_contents.py

```diff
@@ -11,6 +11,9 @@
 )
 from ._framework import ApiTestCase
 
+OWNNER_USER = 'owner@user.com'
+ANOTHER_USER = 'another@user.com'
```

Comment:
Can you just do this without the other user - I would just rely on the default argument to ``_different_user()`` and see if it works? I think the default argument is something that can be set via environment variables and such for testing real servers.

Reply:
Thanks so much for the review!
Actually, I have removed them both, it was the result of playing with the `_different_user()` but as you said is not necessary, and better to leave the default.